### PR TITLE
Update 0.18 axes for release

### PR DESCRIPTION
**Pull request by ajschmidt8**
This PR updates the axes files for the 0.18 release.

### CODE REVIEW

File: ci/axis/rapidsai-base-runtime.yaml [hunk 1]
```diff
@@ -3,15 +3,14 @@ BUILD_IMAGE:
   - rapidsai/rapidsai-nightly
 
 FROM_IMAGE:
-  - rapidsai/rapidsai-core-nightly
+  - rapidsai/rapidsai-core
```
Comment (dillon-cullinan) [inline ci/axis/rapidsai-base-runtime.yaml:6]: We shouldn't need to remove the nightly channels, just exclude.

File: ci/axis/rapidsai-clx-base-runtime.yaml [hunk 2]
```diff
@@ -3,15 +3,14 @@ BUILD_IMAGE:
   - rapidsai/rapidsai-clx-nightly
 
 FROM_IMAGE:
-  - rapidsai/rapidsai-nightly
+  - rapidsai/rapidsai
```
Comment (dillon-cullinan) [inline ci/axis/rapidsai-clx-base-runtime.yaml:6]: Same as above.

File: ci/axis/rapidsai-clx-devel.yaml [hunk 3]
```diff
@@ -3,14 +3,13 @@ BUILD_IMAGE:
   - rapidsai/rapidsai-clx-dev-nightly
 
 FROM_IMAGE:
-  - rapidsai/rapidsai-dev-nightly
+  - rapidsai/rapidsai-dev
```
Comment (dillon-cullinan) [inline ci/axis/rapidsai-clx-devel.yaml:6]: Same as above.

File: ci/axis/rapidsai-devel.yaml [hunk 4]
```diff
@@ -3,14 +3,13 @@ BUILD_IMAGE:
   - rapidsai/rapidsai-dev-nightly
 
 FROM_IMAGE:
-  - rapidsai/rapidsai-core-dev-nightly
+  - rapidsai/rapidsai-core-dev
```
Comment (dillon-cullinan) [inline ci/axis/rapidsai-devel.yaml:6]: Same as above.

**Review by dillon-cullinan — CHANGES_REQUESTED** [review]

File: ci/axis/rapidsai-base-runtime.yaml (hunk 1 above)
Comment (ajschmidt8) [reply ci/axis/rapidsai-base-runtime.yaml:6]: > We shouldn't need to remove the nightly channels, just exclude.

This PR matches how we did it in the `0.17` release (see [main](https://github.com/rapidsai/docker/blob/main/ci/axis/rapidsai-base-runtime.yaml#L5-L6)). Also, there should only ever be one `FROM_IMAGE` when we build these images, so it seems counterintuitive to add a second `FROM_IMAGE` and then exclude the first.

Comment (ajschmidt8) [reply ci/axis/rapidsai-base-runtime.yaml:6]: You may be thinking of the `BUILD_IMAGE` axis value.

Comment (dillon-cullinan) [reply ci/axis/rapidsai-base-runtime.yaml:6]: Ya I got them mixed up.

Comment (dillon-cullinan) [reply ci/axis/rapidsai-base-runtime.yaml:6]: Ya I got them mixed up.

**Review by dillon-cullinan — APPROVED** [review]

**Pull request merged**